.github/workflows/plugin-update.yml

```diff
@@ -0,0 +1,117 @@
+name: Plugin update
+run-name: Update ${{ inputs.repo }} by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        type: string
+        description: 'The owner and repository name as per the github.repository context property.'
+        required: true
```

Comment:
Does it mean it expects the format e.g. `hashicorp/vault-plugin-secrets-kv`?

If yes, what about putting it as an example in the description or split this into 2 inputs (owner & repo) to disambiguate?